### PR TITLE
environment variable to set keep alive timeout

### PR DESCRIPTION
**Pull request by vijayrathore8492**
## Change Summary

To change keep alive timeout with environment variable.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [x] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Review by ketewan — APPROVED** [review]

File: packages/nocodb/docker/index.js [hunk 1]
```diff
@@ -15,7 +15,9 @@ server.set('view engine', 'ejs');
   const httpServer = server.listen(process.env.PORT || 8080, () => {
     console.log(`App started successfully.\nVisit -> ${Noco.dashboardUrl}`);
   })
-  httpServer.keepAliveTimeout = 0; // disable keep-alive
+
+  const keepAliveTimeoutSeconds = parseInt(process.env.NC_KEEP_ALIVE_TIMEOUT_SECONDS);
+  if(keepAliveTimeoutSeconds) httpServer.keepAliveTimeout = keepAliveTimeoutSeconds * 1000;
```
Comment (Faradey27) [inline packages/nocodb/docker/index.js:20]: why do multiply amount seconds to 1000? or does NC_KEEP_ALIVE_TIMEOUT_SECONDS contains ms?

Comment (vijayrathore8492) [reply packages/nocodb/docker/index.js:20]: It contains `seconds` but keepAliveTimeout takes value as `ms`.

Comment (Faradey27) [reply packages/nocodb/docker/index.js:20]: should you than do keepAliveTimeoutSeconds/1000 instead of keepAliveTimeoutSeconds * 1000?

Comment (Faradey27) [reply packages/nocodb/docker/index.js:20]: ah, got it!

**Pull request merged**